test/integration-tests/test-server/test-server.ts

```diff
@@ -27,15 +26,30 @@ const customContext = (context: ExpressContext): Partial<TestContext> => {
     };
 };
 
-export async function startTestServer(): Promise<PolarisServer> {
+const getCustomContext = (
+    context: ExpressContext,
+    customCtx?: Partial<TestContext>,
+): Partial<TestContext> => {
+    if (customCtx) {
+        return customCtx;
+    } else {
+        return customContext(context);
+    }
+};
+
+export async function startTestServer(
```

Comment:
`startTestServer` should not accept these arguments

Reply:
it's optional and i have to pass them here in order to have the ability to create a server with custom context that contains the warnings